### ISSUE TITLE
"draggable: false" with "prevNextButtons: true" triggers "next()" two times 

### ISSUE THREAD

**Issue by tristantbg** [metafizzy/flickity#721]
Can be seen in the official Codepen :
https://codepen.io/desandro/pen/dPdoRO

Is it linked to accessibility or the new staticClick event triggered event on non draggable sliders ?

**Comment by tristantbg** [comment]
Also, the keyboard "accessibility: true" doesn't work either.

Thank you.

**Comment by desandro** [comment]
Thanks for reporting this bug. This looks to a bug in Chrome, but not Firefox or Safari. I'll take a look.

**Comment by bhushanscholastic** [comment]
We are seeing the similar on IE,firefox and Chrome. Do we know when this will be fixed?

Is there any other option to make draggable: false? . We do not want to turn on the draggable to resolve this issue.
